ocaml/xapi/repository_helpers.ml

```diff
@@ -143,7 +143,10 @@ module GuidanceSet = struct
         )
         gs precedences
     in
-    match kind with Recommended -> gs' | Absolute -> remove EvacuateHost gs'
+    if remove_evacuations then
+      remove EvacuateHost gs'
```

Comment:
Why would you remove evacuatehost from absolute guidances? reboothost overrides evacuatehost but it should do that for both recommended and absolute. And if an update really doesn't need evacuation as an absolute guidance then it shouldn't contain it, and if it does XAPI shouldn't remove it.

Reply:
This isn't something that is being changed in this PR.